### PR TITLE
Refactor `constants` to add `constants::load`

### PR DESCRIPTION
**Pull request by therealyingtong**
This makes it easier to load constants into the ECC chip.

### CODE REVIEW

File: src/constants.rs [hunk 1]
```diff
@@ -59,237 +65,181 @@ pub const NUM_WINDOWS_SHORT: usize =
...
+//      1. z + y = u^2,
+//      2. z - y is not a square
+// for the y-coordinate of each fixed-base multiple in each window.
+fn test_zs_and_us<C: CurveAffine>(base: C, z: &[u64], u: &[[[u8; 32]; H]], num_windows: usize) {
+    let window_table = compute_window_table(base, num_windows);
+
+    for ((u, z), window_points) in u.iter().zip(z.iter()).zip(window_table) {
+        for (u, point) in u.iter().zip(window_points.iter()) {
+            let y = *point.coordinates().unwrap().y();
+            let u = C::Base::from_bytes(&u).unwrap();
```
Comment (daira) [inline src/constants.rs:240]:
```suggestion
            let u = C::Base::from_bytes(u).unwrap();
```

File: src/constants/load.rs [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+#[derive(Clone, Debug, Eq, PartialEq)]
+// 8 u's per window
+pub struct WindowUs<F: FieldExt>(pub Box<[F; H]>);
+
+impl<F: FieldExt> From<&[[u8; 32]; H]> for WindowUs<F> {
+    fn from(window_us: &[[u8; 32]; H]) -> Self {
+        Self(
+            window_us
+                .iter()
+                .map(|u| F::from_bytes(&u).unwrap())
```
Comment (daira) [inline src/constants/load.rs:174]:
```suggestion
                .map(|u| F::from_bytes(u).unwrap())
```

**Review by daira — APPROVED** [review]
utACK

**Review by str4d — APPROVED** [review]
utACK

**Review by daira — COMMENTED** [review]
Post-hoc ACK 243f862617be1b72d35a21a71351e66adc911c8d

**Pull request merged**